e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -87,6 +88,62 @@ public async Task RegistryManager_AddDeviceWithTwinWithDeviceCapabilities()
             }
         }
 
+        [LoggedTestMethod]
+        public async Task RegistryManager_BulkLifecycle()
+        {
+            int bulkCount = 50;
+            string[] deviceIds = new string[bulkCount];
```

Comment:
This is not really needed. You can just iterate over devices and get the deviceId from that in the for loop below.

Reply:
True